api4/team.go

```diff
@@ -235,6 +236,60 @@ func getTeamMembersByIds(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(model.TeamMembersToJson(members)))
 }
 
+func addTeamMember(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireTeamId()
+	if c.Err != nil {
+		return
+	}
+
+	var tm *model.TeamMember
+	var err *model.AppError
+	var team *model.Team
+	if team, err = app.GetTeam(c.Params.TeamId); err != nil {
+		c.Err = err
+		return
+	}
```

Comment:
ok, does it mean in the request body of ```client4.go```, I should do like this:
```
func (c *Client4) AddTeamMember(teamId, userId, hash, dataToHash, inviteId string) (*TeamMember, *Response) {
	requestBody := map[string]string{"team_id": teamId}
	...
}
```
if that's the case, the ```member.TeamId == c.Params.TeamId``` may not be necessary as it will always be the same.